docs/advanced-usage.rst

```diff
@@ -49,7 +49,7 @@ an instance:
 This instantiates an object ``bdb`` of class
 :class:`~bigchaindb_driver.BigchainDB`. When instantiating a
 :class:`~bigchaindb_driver.BigchainDB` object without arguments (as above), it
-uses the default BigchainDB API Root Endpoint ``http://localhost:9984/api/v1``.
+uses the default BigchainDB Root URL Endpoint ``http://localhost:9984``.
```

Comment:
I'd prefer "BigchainDB Root URL" (with no "Endpoint" on the end) but I guess this will do.